src/lib/db.ts

```diff
@@ -664,8 +665,21 @@ export class DB {
       },
       timestamp: firebase.database.ServerValue.TIMESTAMP as number,
       content,
-      hidden: false
+      deleted: false
     };
     supportRef.set(support);
   }
+
+  public hideSupport(support: TeacherSupportModelType) {
```

Comment:
I did this differently than something like document visibility, which monitors the model for changes and persists them to Firebase. Since supports are only ever hidden and it just happens in one place, it seemed simpler to just explicitly call a function to update Firebase. Do you have any thoughts on this?